jobs/build/nag-upstream/Jenkinsfile

```diff
@@ -37,44 +37,40 @@ node() {
     // Skip the build for versions in eol/pre-release state
     def out = buildlib.doozer("${doozerOpts} config:read-group --yaml software_lifecycle", [capture: true]).trim()
     def software_lifecycle_phase = readYaml(text: out)["phase"]
-
-    if (software_lifecycle_phase != 'release') {
-        currentBuild.displayName += " [skipped]"
-    } else {
-        timestamps {
-            releaseChannel = slacklib.to(BUILD_VERSION)
-            try {
-                withCredentials([string(credentialsId: 'openshift-bot-token', variable: 'GITHUB_TOKEN')]) {
-                    report = buildlib.doozer("${doozerOpts} images:streams prs list", [capture: true]).trim()
-                    if (report) {
-                        data = readYaml text: report
-                        text = ""
-                        data.each { email, repos ->
-                            text += "*${email} is a contact for these PRs:*\n"
-                            repos.each { _, prs ->
-                                prs.each { pr -> text += ":black_small_square:${pr.pr_url}\n" }
-                            }
-                        }
-                        def attachment = [
-                            color: "#f2c744",
-                            blocks: [[type: "section", text: [type: "mrkdwn", text: text]]]
-                        ]
-                        releaseChannel.pinAttachment(attachment)
-                        if (params.SEND_TO_SLACK) {
-                            releaseChannel.say(":scroll: Howdy! Some alignment prs are still open for ${group}\n", [attachment])
-                        }
-                    } else {
-                        echo "There are no alignment prs left open."
-                        if (params.SEND_TO_SLACK) {
-                            releaseChannel.say(":scroll: All prs are merged for ${group}")
+    def include_master = software_lifecycle_phase == "pre-release" ? '--include-master' : ''
```

Comment:
maybe return if software_lifecyle is not in `[release, pre-release]`. 

Maybe we'll grow an `eol` here, or something else.

Reply:
Makes sense :+1: